frontend/bacmet/contexts/config.ts

```diff
@@ -0,0 +1,13 @@
+import {createContext, useContext} from "react";
+
+export type Config = {
+    apiRoot: string;
+}
+
+export const ConfigContext = createContext<Config>({
+    apiRoot: "http://localhost:5000/api",
+})
```

Comment:
Make it a bit more dynamic:

```suggestion
export const ConfigContext = createContext<Config>({
    apiRoot: `${window.location.protocol}//${window.location.hostname}:5000/api`,
})
```

Reply:
While I kind of agree with you suggestion the fact that the system is partially prebuilt as static html. This means that the `window` element is not available while building the system and thus using this approach is not as straight forward as one would think.

Another problem here  I feel is that it is just a partial solution that is still too locked down to be used outside the scope of our specific development setup. So if possible I would like to leave this to a separate issue for properly setting up a dynamic connection between the systems.

Reply:
I think it might be a good idea to do something like this if we also include the port in the dynamic part but in the current setup that would break the connection.